impl/src/test/java/org/ehcache/impl/internal/loaderwriter/writebehind/PooledExecutorWriteBehindTest.java

```diff
@@ -38,7 +40,7 @@
   }
 
   @Override
-  protected CacheManagerBuilder managerBuilder() {
+  protected CacheManagerBuilder<CacheManager> managerBuilder() {
```

Comment:
Why not fixing the initial declaration of this method?

Reply:
Right. Missed it.